src/SerchCar.py

```diff
@@ -1,15 +1,33 @@
-from ReadCSVFile import ReadCSVFile
+import csv
 
+def search_car(make=None, color=None):
+    with open('cars.csv', 'r') as file:
```

Comment:
Instead of having this line use the function "get_file_data" in the ReadCSVFile.py